### PR TITLE
feat: added video card to homepage

### PR DESCRIPTION
**Pull request by Shubhamkr2610**
Home Page
I have added the Home page . So ignore other files. Focus only on the HomePage.js (in pages ) and Header.js & header.css in component folder in src.

Files to check
src/components/header/header.js
src/components/header/header.css
src/pages/homepage/HomePage.js
src/context/videoContext
PS - Ignore other files. .
image

### CODE REVIEW

**Review by sanjitsarkar — COMMENTED** [review]
Code looks clean and well structured.
Found no issue
Good to merge

**Review by developer-anand — APPROVED** [review]
Looks good to merge, code is easy to understand.
Please add live link also.

File: src/pages/homepage/HomePage.js [hunk 1]
```diff
@@ -1,10 +1,14 @@
...
+import { Sidebar } from '../../components/sidebar/Sidebar';
+import './homepage.css';
+import { VideoCard } from '../../components/videocard/VideoCard';
+import { useVideoList } from "../../context/videoContext";
+
 export const HomePage = () => {
+const {videoList} = useVideoList();
 
+console.log(videoList)
   const handleClick = () =>{
```
Comment (santhoshpemmaka) [inline src/pages/homepage/HomePage.js:11]: If possible remove the console statement. I think you didn't use any statements inside handleClick. remove arrow function.

**Comment by santhoshpemmaka** [comment]
The code looks clean and has a good structure.

Awesome Work @Shubhamkr2610.  Please do the small suggestions.

Good to merge 👍

**Review by santhoshpemmaka — APPROVED** [review]

File: src/pages/homepage/HomePage.js (hunk 1 above)
Comment (Shubhamkr2610) [reply src/pages/homepage/HomePage.js:11]: thanks @santhoshpemmaka  , good pick 
okay I will remove it

**Pull request merged**